src/reducers/targetReducer.js

```diff
@@ -11,6 +12,10 @@ export default function targetReducer(state = initialState, action) {
       return state.setIn(['targetList'], action.targets);
     case actions.LOAD_TARGETS_FAILED:
       return [];
```

Comment:
I think you should return state here, or else you would be changing all the state to [].